jax/_src/scipy/spatial/transform.py

```diff
@@ -0,0 +1,440 @@
+# Copyright 2023 The JAX Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functools
+import re
+import typing
+
+import jax
+import jax.numpy as jnp
+
+
+class Rotation(typing.NamedTuple):
```

Comment:
We can use `@_wraps(scipy.spatial.Rotation)` here to auto-generate the docstring

Reply:
I added this to the class but am unsure if it is working properly? Do I need to also wrap each method in the class? I had trouble finding a similar example... also unsure how to manually build the docs to verify.

Reply:
Added this for `Rotation` and `Slerp` classes.